config/routes/find.rb

```diff
@@ -13,6 +13,7 @@
   get '/privacy', to: 'pages#privacy', as: :privacy
   get '/terms-conditions', to: 'pages#terms', as: :terms
   get '/course/:provider_code/:course_code', to: 'courses#show', as: 'course'
+  get '/placements/:provider_code/:course_code', to: 'courses#placements', as: 'course_placements'
```

Comment:
Similarly to my other comment, is there a reason for not having this as
```suggestion
  get '/course/:provider_code/:course_code/placements', to: 'placements#show', as: 'course_placements'
```